app/models/note.rb

```diff
@@ -2,6 +2,8 @@ class Note < ActiveRecord::Base
   belongs_to :candidate
   belongs_to :user
   before_save :update_candidate
+  validates :comment, presence: true, length: { in: 10..2000 }
```

Comment:
To get the feedback message right: What are we communicating by requiring a 10-character minimum?

“You wrote a short note. There’s a minimum of 10 characters to make sure you write a long-enough note. Can you add more details?"

On Jun 11, 2015, at 7:32 AM, Emily Berkeley <notifications@github.com<mailto:notifications@github.com>> wrote:


In app/models/note.rb<https://github.com/miranmirza/screening-webapp/pull/28#discussion_r32211830>:

> @@ -2,6 +2,8 @@ class Note < ActiveRecord::Base
>    belongs_to :candidate
>    belongs_to :user
>    before_save :update_candidate
> +  validates :comment, presence: true, length: { in: 10..2000 }


My initial thought was to have this in a separate PR, but you're right...this shouldn't go out without the appropriate flash errors; failing silently is the worst.

—
Reply to this email directly or view it on GitHub<https://github.com/miranmirza/screening-webapp/pull/28/files#r32211830>.